content/software/wallets.md

```diff
@@ -32,5 +32,10 @@ interested in trying the new RGB v0.10 features please use the [command-line too
   working on RGB integration and RGB-related tools. Supports both fungible
   and non-fungible assets.
 
+* ### [Bitlight Wallet](https://bitlightlabs.com/wallet)
```

Comment:
BTW I already added Bitlight Wallet here - https://github.com/RGB-WG/rgb.tech/blob/f5eb27268b81baf84f369082be91e93db4920b6c/content/install/wallets.md?plain=1#L30 - but I missed this piece.